machine-learning/tutorials/SentimentAnalysis/Program.cs

```diff
@@ -3,102 +3,131 @@
+            // This is used to format and clean the data.  
+            // Convert the text column to numeric vectors (Features column) 
             // <Snippet7>
-            pipeline.Add(new TextFeaturizer("Features", "SentimentText"));
-            //</Snippet7>
+            var pipeline = new TextFeaturizingEstimator(mlContext,"SentimentText", "Features")
+             //</Snippet7>
 
-            // Adds a FastTreeBinaryClassifier, the decision tree learner for this project, and 
-            // three hyperparameters to be used for tuning decision tree performance.
-            // <Snippet8>
-            pipeline.Add(new FastTreeBinaryClassifier() { NumLeaves = 50, NumTrees = 50, MinDocumentsInLeafs = 20 });
+            // Adds a FastTreeBinaryClassificationTrainer, the decision tree learner for this project  
+            // <Snippet8> 
+                    .Append(mlContext.BinaryClassification.Trainers.FastTree(label: "Label", features: "Features", numLeaves: 50, numTrees: 50, minDatapointsInLeafs: 20));
             // </Snippet8>
 
-            // Train the pipeline based on the dataset that has been loaded, transformed.
+            // Create and train the model based on the dataset that has been loaded, transformed.
             // <Snippet9>
-            PredictionModel<SentimentData, SentimentPrediction> model =
-                pipeline.Train<SentimentData, SentimentPrediction>();
+            Console.WriteLine("=============== Create and Train the Model ===============");
+            var model = pipeline.Fit(dataView);
+            Console.WriteLine("=============== End of training ===============");
+            Console.WriteLine();
             // </Snippet9>
 
-            // Saves the model we trained to a zip file.
-            // <Snippet10>
-            await model.WriteAsync(_modelpath);
-            // </Snippet10>
-
             // Returns the model we trained to use for evaluation.
-            // <Snippet11>
+            // <Snippet10>
             return model;
-            // </Snippet11>
+            // </Snippet10>
         }
 
-        public static void Evaluate(PredictionModel<SentimentData, SentimentPrediction> model)
+        public static void Evaluate(MLContext mlContext, ITransformer model)
         {
-            // Evaluates.
+            // Evaluate the model and show accuracy stats
+            // Load evaluation/test data
+            // <Snippet12>
+            IDataView dataView = _textLoader.Read(new MultiFileSource(_testDataPath));
+            // </Snippet12>
+
+            //Take the data in, make transformations, output the data. 
             // <Snippet13>
-            var testData = new TextLoader(_testDataPath).CreateFrom<SentimentData>();
+            Console.WriteLine("=============== Evaluating Model accuracy with Test data===============");
+            var predictions = model.Transform(dataView);
```

Comment:
>var [](start = 12, length = 3)

What's the docs best practice for var? Leave it as var, or specify the type explicitly for better readability without IDE?